src/providers/frontChat.ts

```diff
@@ -0,0 +1,72 @@
+import { State } from '../types'
+import waitForLoad from '../utils/waitForLoad'
+
+declare global {
+  interface Window {
+    //eslint-disable-next-line @typescript-eslint/no-explicit-any
+    FrontChat: any
+  }
+}
+
+const domain = 'https://chat-assets.frontapp.com'
+/* eslint-disable */
+const loadScript = (onload: () => void) => {
+  if (window.FrontChat) return false
+
+  const script = () => {
+    var s = document.createElement('script')
+    s.type = 'text/javascript'
+    s.async = true
+    s.id = 'front-chat-script'
+    s.src = `${domain}/v1/chat.bundle.js`
+    var x = document.getElementsByTagName('script')[0]
+    x.parentNode?.insertBefore(s, x)
+    s.onload = onload
+  }
+  script()
+
+  return true
+}
+/* eslint-enable */
+
+const load = ({
+  providerKey,
+  setState,
+  beforeInit = () => undefined,
+  onReady = () => undefined
+}: {
+  providerKey: string
+  setState: (state: State) => void
+  beforeInit?: () => void
+  onReady?: () => void
+}): boolean => {
+  const loaded = loadScript(() => {
+    beforeInit()
+
+    // console.log('WINDOW', window)
+    window.FrontChat('init', {
+      chatId: providerKey,
+      useDefaultLauncher: true // optional
```

Comment:
Could you document what the `useDefaultLauncher` attribute does? (Feel free to add a link to documentation here)